e2e/test/iothub/service/IoTHubServiceProxyE2ETests.cs

```diff
@@ -88,17 +88,23 @@ private async Task JobClient_ScheduleAndRunTwinJob(HttpTransportSettings httpTra
                 Tags = new TwinCollection(),
             };
             twin.Tags[JobTestTagName] = JobDeviceId;
-
-            using var jobClient = JobClient.CreateFromConnectionString(s_connectionString, httpTransportSettings);
+            IotHubServiceClientOptions options = new IotHubServiceClientOptions();
+            options.Proxy = httpTransportSettings.Proxy;
+            using var sc = new IotHubServiceClient(s_connectionString, options);
             int tryCount = 0;
             while (true)
             {
                 try
                 {
                     string jobId = "JOBSAMPLE" + Guid.NewGuid().ToString();
                     string query = $"DeviceId IN ['{JobDeviceId}']";
-                    JobResponse createJobResponse = await jobClient
-                        .ScheduleTwinUpdateAsync(jobId, query, twin, DateTime.UtcNow, (long)TimeSpan.FromMinutes(2).TotalSeconds)
+                    ScheduledTwinUpdate twinUpdate = new ScheduledTwinUpdate();
+                    twinUpdate.queryCondition = query;
+                    twinUpdate.twin = twin;
+                    twinUpdate.startTimeUtc = DateTime.UtcNow;
+                    twinUpdate.maxExecutionTimeInSeconds = (long)TimeSpan.FromMinutes(2).TotalSeconds;
```

Comment:
Use an initializer list.

```suggestion
                    ScheduledTwinUpdate twinUpdate = new ScheduledTwinUpdate
                    {
                        queryCondition = query,
                        twin = twin,
                        startTimeUtc = DateTime.UtcNow,
                        maxExecutionTimeInSeconds = (long)TimeSpan.FromMinutes(2).TotalSeconds,
                    }
```